File: IT/Stringhe/Esercizi/Java/Esercizio15.java
/* file Esercizio15.java */

public class Esercizio15 {
	public static void main(String [] args) {
		String s = System.console().readLine("Scrivi una parola (o una frase): ");

		if (s.isEmpty()) {
			System.out.println("Stringa vuota!");
			return;
		}

		/* converte la parola o la frase in caratteri maiuscoli */

		System.out.println(s.toUpperCase());
	}
}

File: IT/1/Stringhe/Esercizi/Python/esercizio15.py
# file esercizio15.py

from __future__ import print_function, unicode_literals
from sys import stdin, version_info
if hasattr(__builtins__, 'raw_input'): input = raw_input

s = input("Scrivi una parola (o una frase): ")
if version_info[0] < 3: s = s.decode(stdin.encoding)

if not s:
	print("Stringa vuota!")
	exit()

# converte la parola o la frase in caratteri maiuscoli

print(s.upper())
